package/src/index.ts

```diff
@@ -0,0 +1,537 @@
+        this._validateArgs({ countryCode })
+        return this._fetch<WebsiteResponse>(`v1/website/${countryCode}`)
+    }
+
+    /**
+	 * Get information about valorant in a region, such as the client version, branch, and server version
+	 * @param region - Region of valorant to get info about
+	 * @returns Information about a regions valorant
+	 */
+    async getVersion(region: Region): Promise<APIResponse<VersionResponse>> {
+        this._validateArgs({ region })
+        return this._fetch<VersionResponse>(`v1/version/${region}`)
+    }
+
+    /**
+	 * Get all of the featured items in the current valorant store
+	 * @returns Featured items in the current valorant store
+	 */
+    async getFeaturedItems(): Promise<APIResponse<FeaturedItemsResponse>> {
+        // ! v2 is bugged for some reason
+        return this._fetch<FeaturedItemsResponse>("v1/store-featured")
+    }
+
+    /**
+	 * Get a list of all in-app-purchases for valorant, such as Radianite and VP
+	 * @returns List of IAPs in valorant
+	 */
+    async getIAPs(): Promise<APIResponse<StoreOffersResponse>> {
+        // ! find out wtf this actually does
+        return this._fetch<StoreOffersResponse>("v1/store-offers")
+    }
+
+    /**
+	 * Will get information about the current maintenances and incidents about a region
+	 * @param region - Region to get info about
+	 * @returns Info about undergoing maintenances and incidents in a region of valorant
+	 */
+    async getStatus(region: Region): Promise<APIResponse<StatusResponse>> {
+        this._validateArgs({ region })
+        return this._fetch<StatusResponse>(`v1/status/${region}`)
+    }
+
+    /**
+	 * Gets raw data for a match from the valorant API. (Not formatted, use only if you know what you are doing)
+	 * @see {@link default.getMatch} for an easier response to use
+	 * @param matchID - The match ID to get details about
+	 * @param region - Region of the match
+	 * @param queries - Extra queries
+	 * @returns Information about the match
+	 */
+    async getRawMatchDetails(matchID: string, region: Region, queries?: any): Promise<APIResponse<RawMatchDetailsResponse>> {
+        this._validateArgs({ matchID, region })
+        return await this._fetch<RawMatchDetailsResponse>("v1/raw", null, {
+            data: { type: "matchdetails", value: matchID, region, queries },
+            method: "POST"
+        })
+    }
+
+    /**
+	 * Gets raw data for a players match history from the valorant API. (Not formatted, use only if you know what you are doing)
+	 * @see {@link default.getMatches} for an easier response to use
+	 * @param puuid - The match ID to get details about
+	 * @param region - Region of the player
+	 * @param queries - Extra queries
+	 * @returns Information about the players match history
+	 */
+    async getRawMatchHistory(puuid: string, region: Region, queries?: any): Promise<APIResponse<RawMatchHistoryResponse>> {
```

Comment:
Why queries any? Should be more explicit.

Reply:
tbh I don't know what queries is. I'll test and update it